app/src/main/java/edu/temple/templetag/TagDetailActivity.java

```diff
@@ -68,7 +70,8 @@ protected void onCreate(Bundle savedInstanceState) {
             tagPop.setText(mTag.getmTagPopularity() + " people are talking about this event");
         }
         tagDesc.setText(mTag.getmTagDescription());
-    
+        tagDesc.setMovementMethod(new ScrollingMovementMethod());
```

Comment:
What is MovementMethod doing?

Reply:
user can scroll through the tag description if its too long